scripts/lint.sh

```diff
@@ -8,3 +8,4 @@ golint                                                                        \
     ./grepip/...                                                              \
     ./cidr2range/...                                                          \
     ./range2cidr/...
+    ./range2ip/...
```

Comment:
See above comment; you need a trailing slash `\` for `range2cidr` since this is a new entry in the list of a single bash statement.